### PR TITLE
MoOnt: add redirect and README

### PR DESCRIPTION
**Pull request by ElisEck**

### CODE REVIEW

File: moont/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+This space is administered by:
+
+**Elisabeth Eckstädt**
+_R&D Engineer_
+[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/)
+Dresden, Germany
+<elisabeth.eckstaedt@eas.iis.fraunhofer.de>
+GitHub: [ElisEck](https://github.com/ElisEck)
+Linkedin: [Elisabeth Eckstädt](https://www.linkedin.com/in/elisabeth-eckst%C3%A4dt-18689115a)
+ORCID: [0000-0003-4396-5307](https://orcid.org/0000-0003-4396-5307)
```
Comment (TallTed) [inline moont/README.md:20]: To preserve desired line-breaks...
```suggestion
**Elisabeth Eckstädt**  
_R&D Engineer_  
[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/)  
Dresden, Germany  
<elisabeth.eckstaedt@eas.iis.fraunhofer.de>  
GitHub: [ElisEck](https://github.com/ElisEck)  
Linkedin: [Elisabeth Eckstädt](https://www.linkedin.com/in/elisabeth-eckst%C3%A4dt-18689115a)  
ORCID: [0000-0003-4396-5307](https://orcid.org/0000-0003-4396-5307)
```

Comment (davidlehn) [reply moont/README.md:20]: This is happening on many PRs.  The trailing spaces seems a bit magic.  Maybe better to suggest bulleted lists?

Comment (TallTed) [reply moont/README.md:20]: @davidlehn — Bullet lists aren't really appropriate for this content. Trailing double (or more) spaces are indeed Markdown magic. The other option is HTML `<br>` tags.

**Comment by dgarijo** [comment]
note that you are only enabling the html serialization so far

**Pull request merged**